dashboard/test/lib/pd/payment/payment_calculator_csf_test.rb

```diff
@@ -12,13 +12,15 @@ class PaymentCalculatorCSFTest < ActiveSupport::TestCase
         create :user_level, user: @qualified_teacher, best_result: ::ActivityConstants::MINIMUM_PASS_RESULT
       end
       @workshop.section.add_student @qualified_teacher
+      create :pd_enrollment, workshop: @workshop, user: @qualified_teacher
 
       # < 10 passing levels: unqualified
       @unqualified_teacher = create :teacher
       9.times do
         create :user_level, user: @unqualified_teacher, best_result: ::ActivityConstants::MINIMUM_PASS_RESULT
       end
       @workshop.section.add_student @unqualified_teacher
+      create :pd_enrollment, workshop: @workshop, user: @unqualified_teacher
```

Comment:
might be worth changing the pd_enrollment factory to create qualified teachers by default